### DISCUSSION TITLE
Browser Memory Limit

### DISCUSSION THREAD

**Discussion by MohammadMahdiJavid** [openwpm/OpenWPM#1077] [Q&A] [answered]
Hi,
are there any specific reasons, that Memory Limit is set to 1.5 GB for each browser?
[Link to Code](https://github.com/openwpm/OpenWPM/blob/ac36c4026dd5b369c3110abf1ec0c92233edf88d/openwpm/task_manager.py#L38)

I mean since we might run multiple browsers (e.g.  8), it will give us a limit of 16 GB, and probably running server has 16 GB, so that's why you might have chosen 1.5 GB?

Thanks

**Comment by vringar** [comment, marked as answer]
Iirc this was the ratio between cores and memory that we had in our servers, so 1,5GB of RAM/core.
But is has been quite a while since I ran large scale crawls so there might be a different optimal ratio.
More ram per browser doesn't have any downsides that I'm aware of. More ram > the profile can grow bigger> fewer restarts due to memory watchdog > better crawl speed
